kernel_patches_daemon/github_sync.py

```diff
@@ -119,7 +119,10 @@ async def get_mapped_branches(self, series: Series) -> List[str]:
                 logging.info(f"Tag '{tag}' mapped to branch order {mapped_branches}")
                 return mapped_branches
 
-        mapped_branches = self.tag_to_branch_mapping["__DEFAULT__"]
+        try:
+            mapped_branches = self.tag_to_branch_mapping["__DEFAULT__"]
+        except KeyError:
+            return []
```

Comment:
You can use `dict.get(key, default)`, e.g:

mapped_branches = self.tag_to_branch_mapping.get("__DEFAULT__", [])